standard/recommendations/core/PER_cache.adoc

```diff
@@ -2,6 +2,6 @@
 [width="90%",cols="2,6a"]
 |===
 ^|*Permission {counter:per-id}* |*/per/core/cache*
-^|A |A WNM MAY specify whether the data should be cached via the `properties.cache` property.
+^|A |A WNM MAY specify that the data should not be cached via the `properties.cache` property.
```

Comment:
`properties.cache` is a boolean, so `true` (default) is also allowed.  Suggest to keep as is.